eagerlib/telegram_bot_api.rb

```diff
@@ -0,0 +1,28 @@
+class TelegramBotApi
+  def initialize(bot_token)
+    raise(ArgumentError, "no bot_token given") unless bot_token
```

Comment:
I wonder if we could just use the keyword argument on the `initialize` method, so we don't need this line